src/js/player.js

```diff
@@ -1856,10 +1856,19 @@ class Player extends Component {
     );
 
     if (!inAllowedEls) {
-      if (this.isFullscreen()) {
-        this.exitFullscreen();
-      } else {
-        this.requestFullscreen();
+      /*
+       * options.dblclick
+       * Set to false to disable double click handling
+       * Set to a function to substitute an external double click handler
+       */
+      if (this.options_.dblclick !== false) {
+        if (typeof this.options_.dblclick === 'function') {
+          this.options_.dblclick(event);
```

Comment:
Should we not allow this to be a function? Then if they want to handle it, they should listen for the event themselves and set this to false.

Reply:
An argument in favor of this being a method is that it makes it much easier to override the default behavior for a particular hotkey/gesture.

Reply:
I think its fine